crates/biome_html_parser/src/syntax/vue.rs

```diff
@@ -0,0 +1,155 @@
+use crate::parser::HtmlParser;
+use crate::syntax::parse_attribute_initializer;
+use crate::syntax::parse_error::expected_attribute;
+use crate::syntax::parse_error::expected_vue_directive_argument;
+use crate::token_source::HtmlLexContext;
+use biome_html_syntax::HtmlSyntaxKind;
+use biome_html_syntax::HtmlSyntaxKind::*;
+use biome_html_syntax::T;
+use biome_parser::Parser;
+use biome_parser::parse_lists::ParseNodeList;
+use biome_parser::parse_recovery::ParseRecoveryTokenSet;
+use biome_parser::parsed_syntax::ParsedSyntax::{Absent, Present};
+use biome_parser::prelude::*;
+
+pub fn parse_vue_directive(p: &mut HtmlParser) -> ParsedSyntax {
```

Comment:
Nit: All functions here don't need to be `pub`. At most, they can be `pub(crate)` where applicable